docs/.vuepress/theme/components/ThemeSwitcher.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <label id="switch" class="switch">
+      <input type="checkbox" v-on:change="toggleTheme" id="slider" :checked="isDarkTheme">
+      <span class="slider round"></span>
+  </label>
+</template>
+
+<script>
+const CLASS_THEME_DARK = 'theme-dark';
+
+const getDomElement = (selector) => {
+  return document.querySelector(selector);
+};
+
+const toggleDarkThemeClassOnHTML = (htmlDomElement, isDarkTheme) => {
+  if (isDarkTheme) {
+    htmlDomElement.classList.add(CLASS_THEME_DARK);
+
+    return;
+  }
+
+  htmlDomElement.classList.remove(CLASS_THEME_DARK);
+};
+
+export default {
+  name: 'ThemeSwitcher',
+  methods: {
+    toggleTheme() {
+      this.isDarkTheme = !this.isDarkTheme;
+      const theme = this.isDarkTheme ? 'dark' : 'light';
+
+      if (localStorage) {
+        localStorage.setItem('theme', theme);
```

Comment:
`localStorage` is domain specific, so I would make the key more specific to avoid collisions with other `handsontable.com` apps. ie. `handsontable/docs::color-scheme`.

Also would move the key to cost to avoid magic string and keep all usages in sync:
```suggestion
        localStorage.setItem(STORAGE_KEY, theme);
```